python/ray/_private/services.py

```diff
@@ -1223,7 +1225,10 @@ def start_raylet(redis_address,
     # The caller must provide a node manager port so that we can correctly
     # populate the command to start a worker.
     assert node_manager_port is not None and node_manager_port != 0
-    config_str = ",".join(["{},{}".format(*kv) for kv in config.items()])
+    config_str = ";".join(["{},{}".format(*kv) for kv in config.items()])
+    assert " " not in config_str, (
```

Comment:
Can you include the config str in the error message? It is hard to debug the cause otherwise.